app/assets/javascripts/discourse/app/components/render-glimmer-container.gjs

```diff
@@ -7,10 +7,7 @@ export default class RenderGlimmerContainer extends Component {
   <template>
     {{~#each this.renderGlimmer._registrations as |info|~}}
       {{~#in-element info.element insertBefore=null~}}
-        <info.component
-          @data={{info.data}}
-          @setWrapperElementAttrs={{info.setWrapperElementAttrs}}
```

Comment:
why is this gone?

Reply:
Oops! That change was supposed to be in DecoratedHtml, not RenderGlimmerContainer

Thanks!